app/furniture/marketplace/locales/en.yml

```diff
@@ -69,6 +69,10 @@ en:
         link_to: "Order History"
       show:
         link_to: "Order %{order_id}"
+    stripe_accounts:
+      new:
+        link_to:
+          "Manage Stripe"
```

Comment:
This seems to be a second `stripe_accounts` section at the same level as the one below?

Reply:
Woops! Good catch!